src/main/java/com/commercetools/sync/inventories/SyncProblemException.java

```diff
@@ -0,0 +1,10 @@
+package com.commercetools.sync.inventories;
+
+/**
+ * Indicates whether sync meets problem which makes further sync unavailable. It may refer to different scope of data,
+ * that are unable to be synced.
+ * e.g. May occur during fetching supply channels what makes all drafts unable to be synced.
+ * e.g.2. May occur during fetching batch of old inventory entries what makes batch of draft unable to be synced.
+ */
+class SyncProblemException extends RuntimeException {
```

Comment:
One more thing:
Handling exceptions in this library is not a trivial concern and most of all it's a separate concern so I wouldn't touch it in this PR. A separate issue should be created for it for the whole library.